packages/account/src/Styles/account.scss

```diff
@@ -1787,17 +1788,27 @@ $MIN_HEIGHT_FLOATING: calc(
             flex-direction: column;
             justify-content: space-between;
         }
+        &__wrapper {
+            padding: 0 2.4rem;
+            border: 1px solid var(--general-section-1);
+            border-radius: $BORDER_RADIUS;
+            margin-top: 1.6rem;
+        }
         &__container {
             display: flex;
-            padding: 0.3rem 0.3rem 0.3rem 1.6rem;
+            padding: 1.1rem 0;
             justify-content: space-between;
             align-items: center;
-            max-width: 26.4rem;
+        }
+        &__container:not(:last-child) {
+            border-bottom: 1px solid var(--general-section-1);
         }
         &__details {
             color: var(--text-prominent);
             font-size: var(--text-size-xs);
             line-height: 1.43;
+            text-align: right;
+            margin-left: 2rem;
```

Comment:
You can remove [L1807 to L1810](https://github.com/binary-com/deriv-app/pull/2973/files#diff-23b1b93d2e9977c91a582c3d639b3d2194cca8f88ef1a12619f3af1c5c107ff1R1807-R1810) and handle the styles from the `Text` component instead.